contracts/escrow/dapps/CompoundFacet.sol

```diff
@@ -92,6 +109,11 @@ contract CompoundFacet is PausableMods, DappMods {
             abi.encodeWithSelector(ICErc20.redeemUnderlying.selector, amount)
         );
 
+        if (tokenAddress == AppStorageLib.store().wrappedNativeToken) {
+            // Wrap ETH back to WETH
+            IWETH(tokenAddress).deposit{ value: amount }();
+        }
+
```

Comment:
move this to the `__compoundRedeem` private function below and also use the `callDapp` function to do so